File: stu_spring/webflux/auth/src/main/java/com/webflux/auth/domain/user/exception/UserNotFoundException.java
package com.webflux.auth.domain.user.exception;

import com.webflux.auth.global.error.exception.ErrorCode;
import com.webflux.auth.global.error.exception.GlobalException;

public class UserNotFoundException extends GlobalException {
    public UserNotFoundException() {
        super(ErrorCode.USER_NOT_FOUND);
    }
}

File: stu_spring/kotlin_basic/src/main/kotlin/com/kotlin/test/kotlintest/domain/user/exceptions/UserNotFoundException.kt
package com.kotlin.test.kotlintest.domain.user.exceptions

import com.kotlin.test.kotlintest.global.exception.GlobalException
import com.kotlin.test.kotlintest.global.exception.enums.ErrorCode

class UserNotFoundException: GlobalException(ErrorCode.USER_NOT_FOUND) {
    companion object {
        @JvmField
        val EXCEPTION = UserNotFoundException()
    }
}